src/main/java/org/snomed/snowstorm/rest/pojo/ItemsPage.java

```diff
@@ -18,6 +18,16 @@
 	private final String searchAfter;
 	private final Object[] searchAfterArray;
 
+	//Default for Jackson (de)serialisation
+	public ItemsPage() {
```

Comment:
I needed a default constructor for `ItemsPage` so Jackson could deserialise the object for my tests.